src/libostree/ostree-repo.c

```diff
@@ -3113,6 +3113,53 @@ reload_remote_config (OstreeRepo          *self,
   return TRUE;
 }
 
+static gboolean
+reload_sysroot_config (OstreeRepo          *self,
+                       GCancellable        *cancellable,
+                       GError             **error)
+{
+  { g_autofree char *bootloader = NULL;
```

Comment:
Why the nested scope here?

Reply:
Ahh I see, in case we have more configs in `[sysroot]`, I guess?

Reply:
Yes - that was it. I was mostly copying the pattern from the `reload_core_config()` and `reload_remtoe_config()` - but also in case we add more to `[sysroot]`.